receiver/otlpreceiver/internal/errors/errors.go

```diff
@@ -40,6 +40,18 @@ func GetHTTPStatusCodeFromStatus(s *status.Status) int {
 	case codes.ResourceExhausted:
 		return http.StatusTooManyRequests
 	// Not Retryable
+	case codes.InvalidArgument:
```

Comment:
@TylerHelmuth maybe we should this function and its tests to the helper i made last time? https://github.com/open-telemetry/opentelemetry-collector/blob/af33ac5acc08dbeb43426f86b526d2cfd29318ab/internal/httphelper/helper.go#L15C6-L15C33

That way the conversions are in the same place

Reply:
@jaronoff97 it is all moving (and properly solved by remembering the actual http status code) via https://github.com/open-telemetry/opentelemetry-collector/pull/9041, so we should stick with that PR so the permanent solution to this issue. This PR fixes the bug for now.

Reply:
cool! I have no problem with that :D